packages/teleterm/src/ui/services/workspacesService/documentsService/documentsService.ts

```diff
@@ -90,17 +89,17 @@ export class DocumentsService {
       leafClusterId: params.leafClusterId,
       kubeId: params.kubeId,
       kubeUri: options.kubeUri,
+      // We prepend the name with `rootClusterId/` to create a kube config
+      // inside this directory. When the user logs out of the cluster,
+      // the entire directory is deleted.
       kubeConfigRelativePath:
         options.kubeConfigRelativePath ||
-        // We prepend the name with `rootClusterId/` to create a kube config
-        // inside this directory. When the user logs out of the cluster,
-        // the entire directory is deleted.
```

Comment:
I moved this comment around because it was messing up syntax highlighting in my editor. ;f